src/mongocxx/test/collection.cpp

```diff
@@ -89,6 +88,8 @@ TEST_CASE("CRUD functionality", "[driver::collection]") {
     }
 
     SECTION("insert and read multiple documents", "[collection]") {
+        coll.delete_many({});
```

Comment:
The additions of delete_many here and below... were those bugs in the tests before? If so, how were we not noticing, or, if not, why does this overall change require this change?